### PR TITLE
Made it last 14 days

### PR DESCRIPTION
**Pull request by blitzwolfz**

### CODE REVIEW

File: src/commands/giveaways.ts [hunk 1]
```diff
@@ -30,7 +30,7 @@ export async function startGiveaway(message: Message, args: string[], client: Cl
         ending = new Date(Date.now() + addTime);
     }
     if (addTime == -1) return message.reply("please specifiy a valid time.")
-    if (addTime > 172800000) return message.reply("You can't start a giveaway that lasts longer than two days!");
+    if (addTime > (20160*60000)) return message.reply("You can't start a giveaway that lasts longer than two days!");
```
Comment (DevilJamJar) [inline src/commands/giveaways.ts:33]: Dude if you’re gonna change it at least change the return message

**Review by DevilJamJar — CHANGES_REQUESTED** [review]

File: src/commands/giveaways.ts (hunk 1 above)
Comment (DevilJamJar) [reply src/commands/giveaways.ts:33]:
```suggestion
    if (addTime > (20160*60000)) return message.reply("You can't start a giveaway that lasts longer than fourteen days!");
```

**Review by DJJ05 — CHANGES_REQUESTED** [review]
Make return message match

**Review by JonathanTheZero — CHANGES_REQUESTED** [review]
What Devil said... please change the return message

**Comment by blitzwolfz** [comment]
ok

**Pull request closed without merging**